edge-modules/edge-dashboard/src/lib.rs

```diff
@@ -229,6 +234,75 @@ fn restart_module(
     Box::new(response)
 }
 
+fn get_logs(
+    req: HttpRequest,
+    context: web::Data<Arc<Context>>,
+    info: web::Query<AuthRequest>,
+) -> Box<dyn Future<Item = HttpResponse, Error = ActixError>> {
+    let module_id = req.match_info().get("id").unwrap_or("");
+    let api_ver = &info.api_version;
+
+    let response = context
+        .edge_config
+        .as_ref()
+        .map(move |config| {
+            let mgmt_uri = config.connect().management_uri();
+            Either::A(
+                Url::parse(&format!("{}/modules/?api-version={}", mgmt_uri, api_ver))
```

Comment:
This url parsing happens almost for every api handler that makes a call to iotedged. Should we make it as a parameter of Context?